tensorflow_probability/python/bijectors/real_nvp_test.py

```diff
@@ -98,11 +98,16 @@ def testBatchedBijectorWithMLPTransform(self):
       self.assertAllClose(ildj_, -fldj_, rtol=1e-6, atol=0.)
 
   def testBijectorConditionKwargs(self):
-    N = 3
-    x_ = np.arange(N * 4 * 2).astype(np.float32).reshape(N, 4 * 2)
+    batch_size = 3
+    x_ = np.arange(batch_size * 4 * 2).astype(np.float32).reshape(
+        batch_size, 4 * 2)
     conditions = {
-        'a': tf.reshape(tf.range(N * 4, dtype=tf.float32), (N, 4)),
-        'b': tf.reshape(tf.range(N * 2, dtype=tf.float32), (N, 2)),
+        'a':
```

Comment:
Sorry to ping again, but can you change these quotations (this line and below) to be double quotes instead of single quotes? I guess the linter doesn't catch that (but the internal one checks if the whole file has the same quotations)